app/models/test_execution.rb

```diff
@@ -46,6 +46,8 @@ def validate_artifact(validators, artifact, options = {})
                              :msg_type => :error, :validator_type => :result_validation, :validator => :smoking_gun)
     end
     execution_errors.only_errors.count > 0 ? fail : pass
+  rescue
+    error
```

Comment:
Might having a _method_ called `error` be confusing for other devs? @okeefm @dehall ?